src/docs/markdown/caddyfile/options.md

```diff
@@ -811,13 +806,29 @@ Also included is the [`proxy_protocol`](/docs/json/apps/http/servers/listener_wr
 			proxy_protocol {
 				timeout 2s
 				allow 192.168.86.1/24 192.168.86.1/24
+				deny 10.0.0.0/8
```

Comment:
This reads like an example. I think it should be changed to the "Syntax" form (like `timeout <duration>` etc) and then move this example below the options list.

Reply:
Better?

Reply:
Did a commit to do what I mean :sweat_smile: